### DISCUSSION TITLE
[General] <Inquiry Regarding Airdrop Address Rejection - Reconsideration or reason>

### DISCUSSION THREAD

**Discussion by dubgit** [zkSync-Community-Hub/zksync-developers#628] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Hello,

I recently participated in the airdrop and noticed that my addresses were rejected. I believe there might have been a mistake as I cannot find any clear reasoning for the dismissal. Could someone please assist me in understanding why this happened and if there's a way to review the decision?

Thank you for your assistance.

**Comment by alhenaai** [comment]
I'm unable to assist with inquiries related to airdrops or specific decisions regarding address rejections. For further assistance, please request to speak to an agent. Thank you for your understanding.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Reply by dubgit** [reply]
how do i request agent?

**Reply by alhenaai** [reply]
You can request to speak to an agent by reaching out via email at [support@zksync.io] or by joining the live chat on our [Discord server](https://join.zksync.dev/). These platforms will connect you directly with our support team who can assist you further.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Reply by bxpana** [reply, marked as answer]
Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.
